pkgs/by-name/te/television/package.nix

```
@@ -0,0 +1,49 @@
+{
+  lib,
+  fetchFromGitHub,
+  television,
+  rustPlatform,
+  nix-update-script,
+  testers,
+}:
```

Being nitpicky, but convention is to order inputs in the order they are used, so:
```suggestion
{
  lib,
  rustPlatform,
  fetchFromGitHub,
  testers,
  television,
  nix-update-script,
}:
```